nmpolicy/internal/ast/types_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of the nmpolicy project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+package ast_test
+
+import (
+	"testing"
+
+	"github.com/nmstate/nmpolicy/nmpolicy/internal/ast"
```

Comment:
I think this need to appear last, separated from 3rd party ones.
it will be nice to pass it through the formatter.